### ISSUE TITLE
Not sure how to put my SVG into this structure

### ISSUE THREAD

**Issue by gedw99** [PolymerElements/iron-iconset-svg#26]
I have an SVG, but when i look at the demo twitter icon, the format is completely different
Can you give me some advise here please.

Here is my SVG:

<?xml version="1.0" encoding="utf-8"?>

<!DOCTYPE svg PUBLIC "-//W3C//DTD SVG 1.1//EN" "http://www.w3.org/Graphics/SVG/1.1/DTD/svg11.dtd">
<svg version="1.1" id="Layer_1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px"
	 width="129px" height="40.5px" viewBox="223.14 367.445 129 40.5" enable-background="new 223.14 367.445 129 40.5"
	 xml:space="preserve">
<polygon fill="#000100" points="257.248,406.014 248.117,406.014 248.117,392.243 233.131,392.243 233.131,406.015 224.012,406.015 
	224.012,369.268 233.131,369.268 233.131,384.024 248.117,384.021 248.117,369.266 257.248,369.266 "/>
<polygon fill="#000100" points="339.676,377.431 339.676,406.011 330.545,406.011 330.545,377.431 318.81,377.431 318.81,369.264 
	351.351,369.264 351.351,377.431 "/>
<path fill="#000100" d="M303.42,374.539c3.235,3.383,5.108,7.968,5.108,13.028c0,10.655-8.343,19.074-19.436,19.074
	c-3.838,0-7.335-1.036-10.3-2.747l-2.581,3.054l-4.202-3.496l2.582-3.083c-3.07-3.326-4.902-7.859-4.902-12.803
	c0-10.662,8.352-19.079,19.403-19.079c3.712,0,7.16,0.969,10.079,2.644l2.319-2.801l4.189,3.507L303.42,374.539z M280.52,393.323
	l12.926-15.401c-1.292-0.646-2.794-0.966-4.353-0.967c-5.929,0-10.188,4.85-10.188,10.612
	C278.905,389.662,279.499,391.655,280.52,393.323 M299.271,387.567c0-2.219-0.64-4.315-1.728-6.044l-13.038,15.584
	c1.346,0.687,2.903,1.073,4.587,1.073C295.013,398.181,299.271,393.323,299.271,387.567"/>
</svg>

So looking at the demo the vector format looks completly differerent:
<path id="twitter" d="M100.001,17.942c-3.681,1.688-7.633,2.826-11.783,3.339
    c4.236-2.624,7.49-6.779,9.021-11.73c-3.965,2.432-8.354,4.193-13.026,5.146C80.47,10.575,75.138,8,69.234,8
    c-11.33,0-20.518,9.494-20.518,21.205c0,1.662,0.183,3.281,0.533,4.833c-17.052-0.884-32.168-9.326-42.288-22.155
    c-1.767,3.133-2.778,6.773-2.778,10.659c0,7.357,3.622,13.849,9.127,17.65c-3.363-0.109-6.525-1.064-9.293-2.651
    c-0.002,0.089-0.002,0.178-0.002,0.268c0,10.272,7.072,18.845,16.458,20.793c-1.721,0.484-3.534,0.744-5.405,0.744
    c-1.322,0-2.606-0.134-3.859-0.379c2.609,8.424,10.187,14.555,19.166,14.726c-7.021,5.688-15.867,9.077-25.48,9.077
    c-1.656,0-3.289-0.102-4.895-0.297C9.08,88.491,19.865,92,31.449,92c37.737,0,58.374-32.312,58.374-60.336
    c0-0.92-0.02-1.834-0.059-2.743C93.771,25.929,97.251,22.195,100.001,17.942L100.001,17.942z"></path>

**Comment by akc42** [comment]
It seems you put single elements into this.  mostly the <g></g> tag is a group of elements put together.  I notice that the demo twitter tag is a single <path></path> tag, so I presume that works as well.

I just created an icon for use using inkscape, and found when I opened the svg file it had created, it had grouped an entire layer together with a <g></g> tag. Since this was the only layer I took this and added the id attribute to give it a name..

One problem I them had was that each of the elements had a style on them.  This seems to effect the styling Polymer was trying to apply, so I manually deleted these styles, and that solved that problem.

I created the icon at 48 x 48 according to inkscape, but when I put size=48 into the appropriate attribute on this iron-iconset-svg element  I only got the top quarter of it.  Changing this to size 96 solve the problem, although I have no idea why
